cmd/config/config.go

```diff
@@ -31,12 +31,15 @@ func NewConfigCmd(analyticsTracker analytics.Tracker) *cobra.Command {
 		Short: "View and modify specific configuration values",
 		Use:   "config",
 		PreRun: func(cmd *cobra.Command, args []string) {
-			analyticsTracker.SendCommandRunEvent(
-				viper.GetString(cliflags.AccessTokenFlag),
-				viper.GetString(cliflags.BaseURIFlag),
-				viper.GetBool(cliflags.AnalyticsOptOut),
-				cmdAnalytics.CmdRunEventProperties(cmd, "config"),
-			)
+			// only track event if there are flags
+			if len(os.Args[1:]) > 1 {
```

Comment:
I'm not sure if there's a better way to do this. Since we define `config --list/--set/--unset` with no action, we only want to send this if we're not showing help.